src/components/Nav/DashboardNav.jsx

```diff
@@ -69,16 +69,14 @@ function DashboardNav(props) {
             : null}
 
           {/* show alert? */}
-          {newInvites.length > 0
```

Comment:
Need to change this back.
App may crash if no invitations.

Reply:
If this crashes it is because of a bug somewhere else, it needs to stay the way I have it.